### PR TITLE
Krovatkin/symbolic shapes3

### PR DESCRIPTION
**Pull request by Krovatkin**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: torch/_decomp/decompositions.py [hunk 1]
```diff
@@ -153,6 +153,12 @@ def hardsigmoid_backward(grad_output: Tensor, self: Tensor):
     )
 
 
+@register_decomposition(aten.hardtanh_)
```
Comment (Krovatkin) [inline torch/_decomp/decompositions.py:156]: is this the right way to do it?

Comment (ezyang) [reply torch/_decomp/decompositions.py:156]: I wouldn't land it this way into master but it's fine for branch

**Comment by ezyang** [comment]
Can you please give these PRs proper titles

File: torch/_decomp/decompositions.py (hunk 1 above)
Comment (Krovatkin) [reply torch/_decomp/decompositions.py:156]: for my education, what should be the proper way to implement this?

Comment (ezyang) [reply torch/_decomp/decompositions.py:156]: We need a consolidated strategy for inplace decomps

**Pull request merged**